chainer/utils/sparse.py

```diff
@@ -0,0 +1,73 @@
+import chainer
+from chainer.backends import cuda
+
+
+class SparseCooMatrix(object):
```

Comment:
Can it be shortened to `CooMatrix`, more similar to scipy's `coo_matrix`?

Reply:
I agree with you. `SparseCooMatrix` is a bit redundant, actually. I will change it to `CooMatrix`.